OPTIONS.md

```diff
@@ -859,9 +859,9 @@ Commands:
   help|h [command] [option]                                                          Display help for commands and options.
   info|i [options]                                                                   Outputs information about your system.
   init|c [generation-path] [options]                                                 Initialize a new webpack project.
-  loader|l [output-path]                                                             Scaffold a loader.
+  loader|l [output-path] [options]                                                             Scaffold a loader.
   migrate|m <config-path> [new-config-path]                                          Migrate a configuration to a new version.
-  plugin|p [output-path]                                                             Scaffold a plugin.
+  plugin|p [output-path] [options]                                                            Scaffold a plugin.
   serve|s [entries...] [options]                                                     Run the webpack dev server.
```

Comment:
Can we fix the alignment as well?

Reply:
We need to set TTY and columns https://github.com/tj/commander.js/blob/master/index.js#L577

Reply:
Let's do it 👍🏻